pkg/disttask/framework/storage/converter.go

```diff
@@ -89,27 +98,16 @@ func row2BasicSubTask(r chunk.Row) *proto.Subtask {
 		Concurrency: int(r.GetInt64(6)),
 		CreateTime:  createTime,
 		Ordinal:     ordinal,
+		StartTime:   startTime,
 	}
 	return subtask
 }
 
 // Row2SubTask converts a row to a subtask.
 func Row2SubTask(r chunk.Row) *proto.Subtask {
 	subtask := row2BasicSubTask(r)
-	// subtask defines start/update time as bigint, to ensure backward compatible,
-	// we keep it that way, and we convert it here.
-	var startTime, updateTime time.Time
-	if !r.IsNull(9) {
-		ts := r.GetInt64(9)
-		startTime = time.Unix(ts, 0)
-	}
-	if !r.IsNull(10) {
-		ts := r.GetInt64(10)
-		updateTime = time.Unix(ts, 0)
-	}
-	subtask.StartTime = startTime
-	subtask.UpdateTime = updateTime
```

Comment:
why remove it, some test need this to check whether it's changed

Reply:
sry... Forgot to push this commit 4a177a2614bf54e2d01f9b4d04a8c5f67e53acf9 😂